doc/WELOTEC.md

```diff
@@ -0,0 +1,18 @@
+# Arrakis-pico-MK3
+
+- **Product Page:** https://www.welotec.com/de/industrie-pcs/
+- **BSP (dynamic layer):** [meta-welotec-bsp](https://github.com/omnect/meta-welotec-bsp)
+- **Boot Media:**
+  - configure BIOS settings to switch between USB and nvme as boot device. 
+  - use [flash-mode 1](../README.md#flash-mode-1) in order to flash an image from USB to nvme
+  - nvme device path is */dev/nvme0n1*
```

Comment:
@mlilien is this correct and enough? do we need an extra chapter in this file to explain how to initially flash nvme?